core/domain/story_jobs_one_off_test.py

```diff
@@ -186,7 +187,7 @@ def _mock_get_story_by_id(unused_story_id):
         # 'successfully migrated' message. Its absence means that the story
         # could not be processed.
         expected = [[u'validation_error',
-                     [u'Story %s failed validation: \'str\' object has '
+                     [u'Story %s failed validation: \'unicode\' object has '
```

Comment:
Will this need to be changed back to `str` after we move to Python 3?

Reply:
Yep

Reply:
Then use `self.assertRegexpMatches(log_msg, 'object has no attribute \'cmd\'')` instead since that's what this test really cares about (not whether it was a str or unicode).

Reply:
We are actually using literal_eval here and comparing lists. So can we still use assertRegexpMatches?

Reply:
We should refactor to iterate through the list.

Reply:
Done